[8.763s] Invoking command in '/home/nvidia/master_ws/src/launch_ws/src/launch': PYTHONPATH=/home/nvidia/master_ws/build/launch/prefix_override:/home/nvidia/master_ws/install/launch/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 -W ignore:setup.py install is deprecated setup.py egg_info --egg-base ../../../../build/launch build --build-base /home/nvidia/master_ws/build/launch/build install --record /home/nvidia/master_ws/build/launch/install.log --install-scripts $base/lib/launch --single-version-externally-managed install_data
[10.500s] Traceback (most recent call last):
[10.502s]   File "<frozen importlib._bootstrap_external>", line 148, in _path_is_mode_type
[10.504s]   File "<frozen importlib._bootstrap_external>", line 142, in _path_stat
[10.506s] FileNotFoundError: [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/zope/__init__.abi3.so'
[10.507s] 
[10.507s] During handling of the above exception, another exception occurred:
[10.507s] 
[10.508s] Traceback (most recent call last):
[10.508s]   File "setup.py", line 3, in <module>
[10.513s]     from setuptools import setup
[10.516s]   File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 17, in <module>
[10.517s]     import setuptools.version
[10.518s]   File "/usr/lib/python3/dist-packages/setuptools/version.py", line 1, in <module>
[10.534s]     import pkg_resources
[10.561s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3254, in <module>
[10.583s]     def _initialize_master_working_set():
[10.583s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3237, in _call_aside
[10.585s]     f(*args, **kwargs)
[10.591s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3279, in _initialize_master_working_set
[10.620s]     tuple(
[10.621s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3280, in <genexpr>
[10.623s]     dist.activate(replace=False)
[10.623s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2784, in activate
[10.623s]     fixup_namespace_packages(self.location)
[10.624s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2296, in fixup_namespace_packages
[10.624s]     subpath = _handle_ns(package, path_item)
[10.625s]   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2202, in _handle_ns
[10.625s]     loader = importer.find_module(packageName)
[10.625s]   File "<frozen importlib._bootstrap_external>", line 546, in _find_module_shim
[10.635s]   File "<frozen importlib._bootstrap_external>", line 1488, in find_loader
[10.638s]   File "<frozen importlib._bootstrap_external>", line 1525, in find_spec
[10.639s]   File "<frozen importlib._bootstrap_external>", line 156, in _path_isfile
[10.639s]   File "<frozen importlib._bootstrap_external>", line 148, in _path_is_mode_type
[10.639s] KeyboardInterrupt
[10.641s] Invoked command in '/home/nvidia/master_ws/src/launch_ws/src/launch' returned '-2': PYTHONPATH=/home/nvidia/master_ws/build/launch/prefix_override:/home/nvidia/master_ws/install/launch/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 -W ignore:setup.py install is deprecated setup.py egg_info --egg-base ../../../../build/launch build --build-base /home/nvidia/master_ws/build/launch/build install --record /home/nvidia/master_ws/build/launch/install.log --install-scripts $base/lib/launch --single-version-externally-managed install_data
